addon/components/frost-button.js

```diff
@@ -227,12 +246,17 @@ export default Component.extend(PropTypeMixin, {
     }
   }),
 
+  // FIXME: jsdoc
```

Comment:
I'm not a fan of putting jsdocs on *every* function, if code is clear enough to be self-documenting then I feel the inclusion of jsdocs actually detracts from readability